sqlalchemy_ibmi/base.py

```diff
@@ -748,8 +748,10 @@ def get_isolation_level(self, dbapi_conn):
     def create_connect_args(self, url):
         opts = url.translate_connect_args(username='user')
         opts.update(url.query)
+        if opts['port'] and opts['port'] is None:
```

Comment:
```suggestion
        if 'port' in opts and opts['port'] is None:
```

You need to use the "in syntax" to check for keys existing. I think that's what you meant here.